lib/logstash/filters/ruby.rb

```diff
@@ -43,7 +43,11 @@ def filter(event,&block)
       @codeblock.call(event,&block)
       filter_matched(event)
     rescue Exception => e
-      @logger.error("Ruby exception occurred: #{e}")
+      err_details = {}
+      err_details["exception"] = e
+      err_details["event"] = event
+      err_details["code"] = @code
```

Comment:
Again, logging the code itself can create a big overhead in logging messages, could we perhaps only log it if the logging level is `trace`?